compiler/p/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -13192,6 +14384,7 @@
    /* .mnemonic    = */ OMR::InstOpCode::vstribl_r,
    /* .name        = */ "vstribl_r",
    /* .description =    "Vector String Isolate Byte Left-Justified Rc=1", */
+   /* .prefix      = */ OMR::Power::InstOpCode::metadata[OMR::InstOpCode::vstribl].prefix,
    /* .opcode      = */ OMR::Power::InstOpCode::metadata[OMR::InstOpCode::vstribl].opcode | 0xbad,
```

Comment:
is it 0xbad any better than the correct x0400?  not we are going to use it anytime soon.

Reply:
This is simply a placeholder, as support for these instructions has not yet been implemented. It is meant to be replaced with the correct value when somebody gets around to actually implementing support for these instructions in the binary encoder.